signature/policy_config_sigstore_test.go

```diff
@@ -157,18 +190,24 @@ func TestNewPRSigstoreSigned(t *testing.T) {
 				},
 			},
 		} {
-			// Special-case this rejected combination:
-			if c.requiresRekor && len(c2.rekorOptions) == 0 {
+			switch {
+			case c.requiresRekor == rekorRequired && len(c2.rekorOptions) == 0: // Special-case this rejected combination
 				continue
```

Comment:
Can’t all of this be just
```go
if (c.requiresRekor == rekorRequired && len(c2.rekorOptions) == 0) ||
   (c.requiresRekor == rekorForbidden && len(c2.rekorOptions) != 0)) {
    continue
}
```
? I think that would process the "no Rekor" entry of `c2`, without having to special-case the test further.